yarn.lock

```diff
@@ -3821,11 +3821,16 @@
   resolved "https://registry.npmjs.org/@okta/odyssey-design-tokens/-/odyssey-design-tokens-1.2.0.tgz#753d981242fd1a295e55e00c8363585b7e329f9b"
   integrity sha512-AmNnCYlfbTUDaOFnEpR+nWljgoFh8uwOfkPwRV1qxv2Zjb7xQZTS02TFQ+kjqH35VpQv8ynbUmCM+fncfXo+IQ==
 
-"@okta/odyssey-design-tokens@^0.17.0", "@okta/odyssey-design-tokens@^0.17.1":
+"@okta/odyssey-design-tokens@^0.17.0":
```

Comment:
We're still getting this dependency shipped with `@okta/odyssey-react-mui-legacy`, which is an alias that Lee set up to point to a 0.x version of Odyssey because most of our existing components use this pre-1.x version since they haven't been migrated yet

Reply:
Ahh, ok tha makes sense! Thank you!